mulesoft/docs-site-ui
# [W-11071448] doc site nav update

ref: W-11071448

# enhancements

- update marketing headers to the latest
- add links to marketing source HTML/CSS/JS for easier references

# bug fix

- update CSS to make mobile menu scrollable 

# how to validate

1. Open terminal, go to this repo, switch to this branch, pull the latest code
2. Run `gulp preview` and open localhost:5252 in the web browser (Chrome, Firefox, Safari)
3. Validate that the header looks like that in mulesoft.com, specifically here are the major changes:
    - Under Product tab, we updated the 'beta' flags next to Flex Gateway and API Governance items to say 'new' and have magenta background color
    - Under Company tab, we are promoting Connect event now on the ad section
4. Decrease the screen size and expand the menu. The menu should be scrollable and you can see all the content
